### DISCUSSION TITLE
媒體業如何用AI人工智慧，實現流量變現？天下雜誌群的經驗談｜投書

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#705] [General]
https://futurecity.cw.com.tw/article/2165
ＡＩ掌控你的生活？聽起來確實有點可怕，但這樣的模式或許已經在你我的身邊正在發生搂，天下文化這樣的方式，除了可以使自己的媒體得到更多的曝光，更多的讀者自己的習慣被摸透了，有可能對讀者看到自己喜歡的東西

**Comment by yjpxi** [comment]
雖然這樣AI 可以根據讀者的數據，為他們推薦個性化的雜誌，推送更加精準的廣告內容，可以讓廣告主更願意支付高價來投放廣告，從而提高廣告收入，但是也帶來了個人隱私被侵犯的風險。
